rascal2/ui/view.py

```diff
@@ -251,6 +288,52 @@ def add_submenus(self, main_menu: QtWidgets.QMenuBar):
         help_menu.addAction(self.open_about_action)
         help_menu.addAction(self.open_help_action)
 
+    def show_or_hide_sliders(self, do_show_sliders=None):
+        """Depending on current state, show or hide sliders for
```

Comment:
Docstring is not in the correct format. Please see [numpydoc format](https://numpydoc.readthedocs.io/en/latest/format.html)